### PR TITLE
Update Code of Conduct to be based on the citizen code of conduct

### PR DESCRIPTION
**Pull request by erinosher**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,98 @@
...
 
-Questions about the Code of Conduct can be directed to info@hacksocnotts.co.uk
+- Violence, threats of violence or violent language directed against another person.
+- Sexist, racist, homophobic, transphobic, ableist or otherwise discriminatory jokes and language.
+- Posting or displaying sexually explicit or violent material.
+ - Posting or threatening to post other people’s personally identifying information ("doxing").
+ - Personal insults, particularly those related to gender, sexual orientation, race, religion, or disability.
+ - Inappropriate photography or recording.
+ - Inappropriate physical contact. You should have someone’s consent before touching them.
+ - Unwelcome sexual attention. This includes, sexualized comments or jokes; inappropriate touching, groping, and unwelcomed sexual advances.
```
Comment (emberdex) [inline README.md]:
```suggestion
 - Unwelcome sexual attention. This includes sexualized comments or jokes, inappropriate touching, groping, and unwelcomed sexual advances.
```

File: README.md [hunk 2]
```diff
@@ -1,21 +1,98 @@
...
+ - Deliberate intimidation, stalking or following (online or in person).
+ - Advocating for, or encouraging, any of the above behavior.
+ - Sustained disruption of societee events, including talks and presentations.
```
Comment (emberdex) [inline README.md]:
```suggestion
 - Sustained disruption of society events, including talks and presentations.
```

File: README.md [hunk 3]
```diff
@@ -1,21 +1,98 @@
...
+
+## 5. Consequences of Unacceptable Behavior
+Unacceptable behavior from any attendee or society member, including sponsors and those with decision-making authority, will not be tolerated.
+
+Anyone asked to stop unacceptable behavior is expected to comply immediately.
+
+If an attendee or society member engages in unacceptable behavior, the committee members, their apointed deputiees, or relevant members of university of union staff may take any action they deem appropriate, up to and including a temporary ban or permanent expulsion from the society without warning (and without refund in the case of a paid event).
+
+## 6. Reporting Guidelines
+If you are subject to or witness unacceptable behavior, or have any other concerns, please notify a committee members, their apointed deputie, or relevant members of university of union staff as soon as possible.
```
Comment (emberdex) [inline README.md]:
```suggestion
If you are subject to or witness unacceptable behavior, or have any other concerns, please notify a committee member, their appointed deputy, or relevant members of university or union staff as soon as possible.
```

File: README.md [hunk 4]
```diff
@@ -1,21 +1,98 @@
...
+
+For incidents that occur outside of a society event you should email the Welfare and Inclusivity Officer (Molly Shalliker <molly.shalliker@hacksocnotts.co.uk>).
+
+For incidents that occur during a society event you should either contact a member of the society commmittee present at the event or email the Welfare and Inclusivity Officer.
```
Comment (emberdex) [inline README.md]:
```suggestion
For incidents that occur during a society event you should either contact a member of the society committee present at the event, or email the Welfare and Inclusivity Officer.
```

File: README.md [hunk 5]
```diff
@@ -1,21 +1,98 @@
...
+For all inquiries relating to the code of conduct, you should contact the Welfare and Inclusivity Officer or a member of the HackSoc Executive Committee.
+
+### Welfare and Inclusivity Officer
+__Molly Shalliker__ (as of September 21st, 2020)
+[molly.shalliker@hacksocnotts.co.uk](mailto:molly.shalliker@hacksocnotts.co.uk)
+
+### HackSoc Executive Commitee
+[exec@hacksocnotts.co.uk](mailto:exec@hacksocnotts.co.uk)
+
+The latest contact details, including Discord Usernames can also be found in the [#rules](https://discord.com/channels/500363419732410368/500366608448094238) channel on the HackSoc Discord Server.
```
Comment (emberdex) [inline README.md]:
```suggestion
The latest contact details, including Discord usernames can also be found in the [#rules](https://discord.com/channels/500363419732410368/500366608448094238) channel on the HackSoc Discord server.
```

File: README.md [hunk 6]
```diff
@@ -1,21 +1,98 @@
...
+
+## 8. Scope
+We expect all members and attendees (contributors, paid or otherwise; sponsors; and other guests) to abide by this Code of Conduct in all society venues–online and in-person–as well as in all one-on-one communications pertaining to society business.
+
+This code of conduct and its related procedures also applies to unacceptable behavior occuring outside the scope of society activities when such behavior has the potential to adversely affect the safety and well-being of society members.
+
+## 9. Other Applicable Policies
+As HackSoc is a University of Nottingham Students' Union Society and Operates at the University of Nottingham, in addition to our code of conduct, you are also expect to comply with the [University of Nottingham Students' Union's Student Groups Code of Conduct](https://af602652235155630226-3e0448b2dcbed5ff06b014334489e811.ssl.cf3.rackcdn.com/code-of-conduct-guidance-document-october-2019.pdf) and the [University of Nottingham's Code of Discipline](https://www.nottingham.ac.uk/governance/documents/code-of-discipline-010120.pdf).
```
Comment (emberdex) [inline README.md]:
```suggestion
As HackSoc is a University of Nottingham Students' Union Society and Operates at the University of Nottingham, in addition to our code of conduct, you are also expected to comply with the [University of Nottingham Students' Union's Student Groups Code of Conduct](https://af602652235155630226-3e0448b2dcbed5ff06b014334489e811.ssl.cf3.rackcdn.com/code-of-conduct-guidance-document-october-2019.pdf) and the [University of Nottingham's Code of Discipline](https://www.nottingham.ac.uk/governance/documents/code-of-discipline-010120.pdf).
```

**Review by Fraetor — COMMENTED** [review]
Line 13 should it be "their actions" rather than "our actions"?

**Review by Fraetor — DISMISSED** [review]
Line 86, replace "fo" with "of"
Also how do you do the suggested change thing?

File: README.md [hunk 7]
```diff
@@ -1,21 +1,98 @@
...
+__Molly Shalliker__ (as of September 21st, 2020)
+[molly.shalliker@hacksocnotts.co.uk](mailto:molly.shalliker@hacksocnotts.co.uk)
+
+### HackSoc Executive Commitee
+[exec@hacksocnotts.co.uk](mailto:exec@hacksocnotts.co.uk)
+
+The latest contact details, including Discord Usernames can also be found in the [#rules](https://discord.com/channels/500363419732410368/500366608448094238) channel on the HackSoc Discord Server.
+
+## 11. License and attribution
+This Code of Conduct is adapted from the Citizen Code fo Condcut
```
Comment (emberdex) [inline README.md]:
```suggestion
This Code of Conduct is adapted from the Citizen Code of Conduct
```

**Review by AksanDotDev — APPROVED** [review]

**Review by Kairosite — APPROVED** [review]

**Pull request merged**